### PR TITLE
Renamed to staticfile and fixed test cases

### PR DESCRIPTION
**Pull request by gsingh93**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,19 +32,19 @@ It works well in combination with the [mounting handler][mounting-handler].
...
 
-If you're using a `Cargo.toml` to manage dependencies, just add the `static` package to the toml:
+If you're using a `Cargo.toml` to manage dependencies, just add the `staticfile` package to the toml:
 
 ```toml
-[dependencies.static]
+[dependencies.staticfile]
 
-git = "https://github.com/iron/static.git"
+git = "https://github.com/iron/staticfile.git"
```
Comment (reem) [inline README.md]: can you change this to recommend a crates.io dependency?

File: README.md [hunk 2]
```diff
@@ -13,20 +13,18 @@ let mut mount = Mount::new();
...
 ```
 
-Note that `static` is a reserved keyword, so the crate will need to be imported as `extern crate "static" as static_file;`.
-
 See [`examples/doc_server.rs`](examples/doc_server.rs) for a complete example that you can compile.
 
 ## Overview
 
-static is a part of Iron's [core bundle](https://github.com/iron/core).
+staticfile is a part of Iron's [core bundle](https://github.com/iron/core).
```
Comment (reem) [inline README.md]: Just remove this line, that repo doesn't exist anymore and common is going to be totally redone anyway.

**Comment by reem** [comment]
@gsingh93 left some nits, everything else looks great, thanks for having so much patience here.

**Comment by gsingh93** [comment]
Updated.

**Comment by reem** [comment]
Woo!

**Comment by Hoverbear** [comment]
Woo!

**Pull request merged**